app/(dashboard)/mailboxes/[mailbox_slug]/[category]/conversation/conversationContent.tsx

```
@@ -0,0 +1,406 @@
+import FileSaver from "file-saver";
+import {
+    ArrowUp,
+    Download,
+    X,
+} from "lucide-react";
```

ESLint

[error] Replace `⏎····ArrowUp,⏎····Download,⏎····X,⏎` with `·ArrowUp,·Download,·X·`

(prettier/prettier)